terraform/deployments/govuk-publishing-platform/dns.tf

```diff
@@ -31,8 +27,6 @@ resource "aws_route53_zone" "internal_private" {
 resource "aws_acm_certificate" "workspace_public" {
   domain_name = "*.${local.workspace_external_domain}"
 
-  subject_alternative_names = local.is_default_workspace ? ["*.${local.workspace}.${var.publishing_service_domain}"] : null
```

Comment:
can you put this line back, just remove the `${local.workspace}`, i.e.
`subject_alternative_names = local.is_default_workspace ? ["*.${var.publishing_service_domain}"] : null`

Reply:
Isn't it covered in https://github.com/alphagov/govuk-infrastructure/blob/integration/terraform/deployments/govuk-publishing-platform/origins_certificates.tf ?

Reply:
no, the workspace_public cert is used for the ALB of the origin and passed to the `origin` module. e.g. https://github.com/alphagov/govuk-infrastructure/blob/e87dbc0412b06bce6ca23891f8c859fc58c92d32/terraform/deployments/govuk-publishing-platform/frontends_origins.tf#L62

Reply:
Ok got it. Fixed as suggested